backend/api/swagger/experiment.swagger.json

```diff
@@ -17,7 +17,7 @@
   "paths": {
     "/apis/v1beta1/experiments": {
       "get": {
-        "summary": "Find all experiments.",
+        "summary": "Find all experiments. Support pagination and sorting on certain fields.",
```

Comment:
Support pagination and -> Supports pagination, and

Reply:
done